ablNeutralEdge/setup.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env
+"""
+Run this to generate the input files
+You will need aprepro in the path when you execute
+
+'python setup.py' will generate the ablNeutralEdge cases
+'python setup.py /absolute/path/to/build/dir' will
+generate the actuatorLine cases
+"""
+import sys
+import os
+import subprocess
+
+mesh_size = ["02", "05", "10", "20", "40"]
+time_step = [0.15,  0.3,  0.6,  1.2,  2.4]
+cases = []
+
+input_file = "input.yaml"
+output_file = "{base}_{ms}m_sgs2_CFL_485.yaml"
+
+if len(sys.argv) > 1:
```

Comment:
I would feel a bit more comfortable if we check that this argument is a string to a valid directory with something like:
```
import os

use_actuator = False
if len(sys.argv) > 1:
    if os.path.isdir(sys.argv[1]):
        use_actuator = True
    else:
        # technically we should also check that nrel5mw_base_dir is a valid and that the ".dat" file exist
        print("The argument used is not a valid path")
        return 1
else:
    use_actuator = False
```

Reply:
These are all good suggestions.  I admit the interface is pretty clunky at this point.  I'll clean it up.